### ISSUE TITLE
[REQUEST]Gundam Breaker 3 - PCSH00207

### ISSUE THREAD

**Issue by rashraph** [Electry/VitaGrafix#108]
Hi. Thank you for sharing this super plugin for it just bring new life to games on psvita.

I humbly request a native resolution and fps increasement mod for Gundam Breaker 3 [Asia version]-PCSH00207. Here is the eboot file: 
[PCSH00207.zip](https://github.com/Electry/VitaGrafix/files/2453652/PCSH00207.zip)

Thank you !!

**Comment by FormerNoob** [comment]
When improving the resolution, it crash.
But it work at a lower resolution, such as  480x272.
I think its oringinal  resolution is 720x408, 
	0x0047FA82: 0x7934F45F '_.4y' - movs.w     v6, #0x2d0	5FF43479
	0x0047FA86: 0x78CCF45F '_..x' - movs.w     v5, #0x198	5FF4CC78
[PCSH00207][ASIA][v1.30][DLC][Mai233]
Why?Is the Vmemory not enough?@Electry

**Comment by Electry** [comment]
Yeah it does crash, but I don't really know why. I don't think it is vram.
Interestingly, 720x412 works, but 720x420 does not.

**Comment by eddyfire** [comment]
Hi @electry and @ FormerNoob, 

Found this translatation on the chinese website. Ut its for gundam breaker 2. U think u could try it?

Translation :
At first I thought it was the reason for the lack of memory, but I still allocated 8MB of video memory and still reported an error, indicating that this is not the reason, it may be memory. Insufficient reason. Then I allocated 3MB of memory in a memory copy function and finally succeeded. 
Corresponding version is up to the destroyer 2 Hong Kong version 1.03 [PCSH00132] 
in config.txt, by adding 
[PCSH00132] 
IB = 960x544 
Add patchlist.txt in 
# GUNDAM BREAKER 2 [ASIA 1.03] 
[PCSH00132, Eboot.bin, 0x52E6D297] 
@ IB 
0:0x479446 t2_mov(1,1,0x900000) 
0:0x32FAD6 t2_mov(1,4,<ib_w>) 
0:0x32FADA t2_mov(1,5,<ib_h>)

**Comment by eddyfire** [comment]
Hi @Electry any news for this? was hoping it could be done.. We believe in you

**Comment by hasturfenric** [comment]
what about Gundam Breaker 3 Break Edition released in Asia that's in English?

**Comment by Dhoni777x** [comment]
![IMG_20230328_132234](https://user-images.githubusercontent.com/129149570/228151086-f1c6c39e-001b-433f-8e1c-821f04975fe4.jpg)
how to solve it how this can not work?
